package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
How is that "safe"? Is there any reason we don't always want the __latest__ version of `jsonlint`? I can't see any.